cloudformation/stacks/AutoSpotting/template.yaml

```diff
@@ -40,6 +40,17 @@
         "Instance termination method.  Must be one of 'autoscaling' (default) or
         'detach' (compatibility mode, not recommended)"
       Type: "String"
+    TerminationNotificationAction:
+      AllowedValues:
+        - "auto"
+        - "detach"
+        - "terminate"
+      Default: "terminate"
```

Comment:
I would set the default to "auto"

Reply:
Sure, that is the right one!
Just forgot to make the change.